scripts/release-binary.sh

```diff
@@ -152,6 +152,8 @@ do
   BINARY_NAME="${HOME}/${BINARY_BASE_NAME}-${SHA}.tar.gz"
   SHA256_NAME="${HOME}/${BINARY_BASE_NAME}-${SHA}.sha256"
   # shellcheck disable=SC2086
+  bazel build ${BAZEL_BUILD_ARGS} ${CONFIG_PARAMS} @com_googlesource_chromium_v8//:build
```

Comment:
Can you add a comment about this separated step?

Reply:
good point